Sources/Build/BuildPlan.swift

```diff
@@ -2272,8 +2267,23 @@ private func generateResourceInfoPlist(
     return true
 }
 
-fileprivate extension TSCUtility.Triple {
-    var isSupportingStaticStdlib: Bool {
+extension BuildParameters {
+    fileprivate var shouldStaticallyLinkSwiftRuntimeLibrary: Bool {
+        guard !self.disableAutomaticSwiftRuntimeStaticLinking else {
+            return false
+        }
+        guard self.triple.isSupportingStaticStdlib else {
+            return false
+        }
+        guard self.configuration == .release else {
+            return false
+        }
+        return true
+    }
```

Comment:
Should we do it in one guard statement?
```suggestion
        guard !disableAutomaticSwiftRuntimeStaticLinking,
              triple.isSupportingStaticStdlib,
              configuration == .release else {
                return false
            }
        return true
```

Reply:
it def could, but i find it harder to read when it's more than 2, so prefer to spell it out long form

Reply:
Got it.